backend/hdf5/FileHDF5.cpp

```diff
@@ -371,6 +385,14 @@ bool FileHDF5::checkHeader(FileMode mode, bool throw_error) {
         check = false;
         message << "File is not a valid NIX file, version attribute missing!";
     }
+
+    if (check && my_version >= file_format_version) {
```

Comment:
```suggestion
    if (check && my_version >= nix::FormatVersion({1, 2, 0})) {
```

We should check against `1.2.0`, no?  The version that introduced the ID.

Reply:
good catch, but actually it should be 
```file_format_version >= nix::FormatVersion({1, 2, 0})```
correct?